lib/identity_config.rb

```diff
@@ -155,8 +155,10 @@ def self.build_store(config_map)
     config.add(:lexisnexis_trueid_account_id, type: :string)
     config.add(:lexisnexis_trueid_username, type: :string)
     config.add(:lexisnexis_trueid_password, type: :string)
-    config.add(:lexisnexis_trueid_liveness_workflow, type: :string)
-    config.add(:lexisnexis_trueid_noliveness_workflow, type: :string)
+    config.add(:lexisnexis_trueid_liveness_cropping_workflow, type: :string)
+    config.add(:lexisnexis_trueid_liveness_nocropping_workflow, type: :string)
+    config.add(:lexisnexis_trueid_noliveness_cropping_workflow, type: :string)
+    config.add(:lexisnexis_trueid_noliveness_nocropping_workflow, type: :string)
```

Comment:
re: the note about modifying the configs to go with the deploy... we could also add ruby-level aliases if wanted to simplify the transition?

```ruby
class << IdentityConfig.store
  alias_method :old_name, :new_name
end
```

Reply:
> re: the note about modifying the configs to go with the deploy... we could also add ruby-level aliases if wanted to simplify the transition?

That's an interesting idea! 🤔  Though I'd still feel it may be smoother to juggle redundant names in the configs temporarily than it would be to have them in the code?

Reply:
I don't like futzing with the config if we already have values in deployed envs, but if we don't have any yet it's easier to not alias. At the end of the day it's six of one, half a dozen of the other, happy to go with whatever you pick

Reply:
Looking at this again, I'm not actually sure what scenario the alias would help with, since the new code would always point to the new configuration values anyways. Getting the new configs into the deployed environments just makes sure they're there for when the new code is deployed, after which we can remove the old configs. I might be missing something though?

Reply:
No I think I was assuming there were already values set that we care about that we wanted to preserve --- your way is definitely easier